src/vuex/modules/session.js

```diff
@@ -101,6 +105,21 @@ export default () => {
       commit(`setSignIn`, true)
       commit(`setSessionType`, sessionType)
       commit(`setUserAddress`, address)
+
+      // Add sign in address to addresses array if not exist previously
+      let session = {
```

Comment:
I think this part could be refactored out to reduce complexity of the signIn function.
Also: Does this store the addresses somewhere? So when I refresh the page, are the past addresses still there?

Reply:
You'r right! It doesn't store the addresses, I get with it ...

Reply:
> I think this part could be refactored out to reduce complexity of the signIn function.
> Also: Does this store the addresses somewhere? So when I refresh the page, are the past addresses still there?

What do you think we can refactor?

Reply:
It could be refactored in an action `rememberAddress({ state, commit }, address)`

Reply:
- Now it saves used addreses in local storage, so it persists across page reloads.
- Refactor done as action `async rememberAddress({ state, commit }, { address, sessionType })`